### DISCUSSION TITLE
How to use runtime variables from env on NextJS 13

### DISCUSSION THREAD

**Discussion by volnei** [vercel/next.js#46933] [Help]
### Summary

I need to use runtime variable, based on environment no kubernentes. But I've noticed that variables are replaced by current value from current env at build time.

**Comment by richardokonicha** [comment]
Bumping this request up, the next team had something to say about this but did not give further instructions on how best to achieve this. 
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

<img width="813" alt="Screenshot 2023-08-25 at 13 52 44" src="https://github.com/vercel/next.js/assets/48168290/4e5e4d95-b011-47d5-bf86-20337f44f57d">
